.codecov.yml

```diff
@@ -6,15 +6,15 @@ component_management:
     - component_id: crashtracker  # this is an identifier that should not be changed
       name: crashtracker  # this is a display name, and can be changed freely
       paths:
-        - crashtracker
```

Comment:
Could there be a linter rule on naming?

Reply:
I looked around for a little bit and couldn't find a linter rule for anything like this. We'd have to roll our own check, most likely.